app/src/main/java/com/andrew/tasky/domain/AgendaItem.kt

```diff
@@ -12,5 +13,6 @@ data class AgendaItem(
     val description: String,
     val startDateAndTime: LocalDateTime,
     val endDateAndTime: LocalDateTime? = null,
-    val reminderTime: ReminderTimes
+    val reminderTime: ReminderTimes,
+    val photos: List<Uri>? = null
```

Comment:
As I said in WhatsApp, the idea of just putting plain Uris in a list is a bit simplified. What you need here is a way to distinguish between local (represented by a Uri) and remote photos (represented by a URL), since when you're updating an event, you only want to upload the new ones of course.